internal/patroni/config.go

```diff
@@ -152,6 +152,20 @@ func clusterYAML(
 		},
 	}
 
+	// if a Patroni log file size is configured, configure volume file storage
+	if patroniLogFileSize != 0 {
+
+		root["log"] = map[string]any{
+			// Configure the Patroni log settings
+			// - https://patroni.readthedocs.io/en/latest/yaml_configuration.html#log
+			"dir":       naming.PatroniPGDataLogPath,
+			"file_num":  1,
+			"file_size": patroniLogFileSize,
```

Comment:
❓ Does `file_num` go into [RotatingFileHandler](https://docs.python.org/3/library/logging.handlers.html#logging.handlers.RotatingFileHandler).backupCount? If so, my understanding is that there will be two files of `file_size` bytes, `patroni.log` and `patroni.log.1`.

Reply:
Good catch, that looks right, which is not what I intended. I'm thinking that means I'll want to set that value to `0` instead. I'll test to make sure that works as expected.

Reply:
It doesn't.... Apparently to get the file rotation behavior, `file_num` needs to be >= 1
https://github.com/python/cpython/blob/3.11/Lib/logging/handlers.py#L134
If set to zero, the file just grows continually. An easy solution would be to divide the size given by two, leave the `file_size` setting at 1 and update the name (as you mention below) so that it's clear what we're doing. Thoughts on that approach?

Reply:
div 2 sounds good